roles/press_worker/templates/etc/supervisor/conf.d/press_celery_workers.conf

```diff
@@ -0,0 +1,12 @@
+{% for i in range(0, hostvars[inventory_hostname].press_worker_count|default(1), 1) %}
```

Comment:
Isn't a default of 1 a little small? Especially if the tasks can take a really long time.

Reply:
None of the currently implemented task will take a long time. Shouldn't take but 1-2sec max.